frontend/src/styles/AppFooter.css

```diff
@@ -1,3 +1,16 @@
 .AppFooter {
     /* Styles go here */
-}
\ No newline at end of file
+    background-color: #7FC1AC;
```

Comment:
Colors should be added to App.css as root variables.

Reply:
Resolved by 72b479c

Reply:
added as your suggestion